sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/SetLocaleForDataTypes.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Data;
+using System.Globalization;
+
+namespace Tests.Diagnostics
```

Comment:
The check to the culture is not required by this rule (maybe we could extend at some point).